api/scpca_portal/models/sample.py

```diff
@@ -120,6 +120,33 @@ def bulk_create_from_dicts(
 
         Sample.objects.bulk_create(samples)
 
+    def get_metadata(self) -> Dict:
+        sample_metadata = {
+            "scpca_sample_id": self.scpca_id,
+        }
+
+        included_sample_attributes = [
+            "age_at_diagnosis",
+            "diagnosis",
+            "disease_timing",
+            "sex",
+            "subdiagnosis",
+            "tissue_location",
+            "includes_anndata",
+            "is_cell_line",
+            "is_xenograft",
+            "sample_cell_count_estimate",
+        ]
+        sample_metadata.update(
+            {key: getattr(self, key) for key in dict(self) if key in included_sample_attributes}
+        )
+
+        sample_metadata.update(
+            {key: self.additional_metadata[key] for key in self.additional_metadata}
+        )
+
+        return sample_metadata
+
```

Comment:
When you write a ticket about updating how we handle sample metdata / moving away from `sample.additional_metadata` and towards `sample.metadata`. Include this method as a reason why.

Just as a reminder we want to move toward thinking about these attribtues as one of the following:
- metadata (raw inputs from s3)
- attributes (denormalized metadata keys)
- aggregate values (populated after we do bulk create of a related model)

Then we can think about this data in two contexts: persistence / querying and presentation.

This function should be excluded keys (if any) in the same way we do that for `Library.get_metadata`

Reply:
Here is the described issue: https://github.com/AlexsLemonade/scpca-portal/issues/774